src/commands/coffeechats/coffeeGenerate.ts

```diff
@@ -0,0 +1,24 @@
+import { Message } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { generateFutureMatches } from '../../components/coffeechat';
+import { AdminCommand } from '../../utils/commands';
+
+class coffeeSignupCommand extends AdminCommand {
```

Comment:
Rename class. Also how do we trigger this generate command when a user gets/removes the coffee chat role?

Reply:
match validation is ran every coffeeMatch to check if matches are still valid for whoever still wants to coffeechat.
I'll note that with comments